engine/src/test/scala/cromwell/engine/WorkflowStoreSpec.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.engine
+
+import cromwell.engine.workflow.WorkflowStore
+import cromwell.util.SampleWdl.HelloWorld
+import org.scalatest.{FlatSpec, Matchers}
+
+class WorkflowStoreSpec extends FlatSpec with Matchers with WorkflowStore {
```

Comment:
It's kind of strange that this extends the `WorkflowStore` trait directly, and makes it so these "unit" tests are cross-talking.